test/python/opflow/test_gradients.py

```diff
@@ -1197,6 +1279,29 @@ def test_qfi_simple(self, method):
             actual = qfi.assign_parameters(value_dict).eval()
             np.testing.assert_array_almost_equal(actual, correct_values[i], decimal=1)
 
+    def test_QFI_phase_fix(self):
```

Comment:
I think function names are strictly lowercase 🙂 
```suggestion
    def test_qfi_phase_fix(self):
```